guides/_subsections/create-an-automated-rule.md

```diff
@@ -112,64 +112,64 @@ target.annotations.cryostat.PORT > 3000 && target.annotations.platform[‘io.kub
   </li>
   <li>
       {% include howto_step.html
-        summary="Check your match expression"
+        summary="Check your <code>Match Expression</code>"
         image-name="2.3.0/create-an-automated-rule-3.png"
-        caption="You can select a target JVM to view its properties and use them to build your Match Expression."
+        caption="You can select a target <b>JVM</b> to view its properties and use them to build your <code>Match Expression</code>."
         text="
           <p>
-          When you enter a match expression in the <i>Match Expression</i> field, the <i>Match Expression Visualizer</i> will highlight which targets are matched.
+          When you enter a <code>Match Expression</code> in the <i>Match Expression</i> field, the <i>Match Expression Visualizer</i> will highlight which targets are matched.
           </p>
           "
       %}
   </li>
   <li>
       {% include howto_step.html
-        summary="<i>(Optional)</i> Adjust rule parameters"
+        summary="<i>(Optional)</i> Adjust Rule Parameters"
```

Comment:
I think <i>Rule Parameters </i> should be italicized because they're parameters that the user sets themselves.

Reply:
Same in line 30